infra/modules/cicd/main.tf

```diff
@@ -80,43 +116,134 @@ resource "google_cloudbuild_trigger" "app_new_build" {
       invert_regex = false
     }
   }
-  substitutions = {
-    _AR_REPO = "${google_artifact_registry_repository.default.location}-docker.pkg.dev/${google_artifact_registry_repository.default.project}/${google_artifact_registry_repository.default.name}/app"
+  build {
+    images = ["${local.artifact_registry_repo}/app:$${SHORT_SHA}"]
```

Comment:
Needs a tweak. This is printing out `${SHORT_SHA}` rather than replacing it. 

<img width="595" alt="Screenshot 2023-04-20 at 5 18 54 PM" src="https://user-images.githubusercontent.com/1331216/233513293-0d2a7b9b-0383-4b5f-a9da-cb53f6c3dfab.png">

Reply:
Yep this is correct, we do want it to sub in the built-in variable `$SHORT_SHA` when it runs.  You'll see the correct value in the build log